<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="IPv4Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose IPv4 adress:</string>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose PORT:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPv4LineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>211</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>191</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>ClientWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
